prog/main.go

```diff
@@ -259,6 +261,8 @@ func main() {
 
 	flag.Parse()
 
+	app.InitializeTopologies()
```

Comment:
Instead of using `InitializeTopologies()` I would leave the `init()` from `api_topologies.go` as it was and instead incorporate a function to add a container filter (e.g. `AddTopologyOptions(id string, options ...APITopologyOption)` ) to the topologies registry. Then you can simply invoke `AddTopologyOptions()` from here.